packages/config/src/validate/validations.js

```diff
@@ -91,6 +91,14 @@ const POST_NORMALIZE_VALIDATIONS = [
     message: 'must be a string.',
     example: () => ({ plugins: [{ package: 'netlify-plugin-one' }] }),
   },
+  // We don't allow `package@tag|version` nor `git:...`, `github:...`,
+  // `https://...`, etc.
+  {
+    property: 'plugins.*.package',
+    check: package => !package.includes('@') && !package.includes(':'),
```

Comment:
This will not allow:
```toml
[[plugins]]
  package = "@sentry/netlify-build-plugin"
```
Right?

Could we use https://github.com/npm/validate-npm-package-name? Probably we'll need to run it on non local plugins only

Reply:
Good catch! I have fixed it to make it work with local plugins and with plugins with a npm `@scope`.

Looking at the [logic of `validate-npm-package-name`](https://github.com/npm/validate-npm-package-name/blob/master/index.js), it looks like this is mostly about validating that the package name is valid to create a new package. In our case, the plugin has already been created. What we want instead is ensure users are not appending a `@version` to it, or using a repository URL. Do you think we should still use it as a fallback?

Reply:
Just wondering if we might have missed something in the validation that can be covered by that lib, for example if a user makes a typo and uses an upper case letter - would that break gracefully?
Don't think it's a blocker for the current implementation

Reply:
Makes sense. Fixed!